pkg/virt-controller/watch/migration.go

```diff
@@ -458,6 +458,14 @@ func (c *MigrationController) updateStatus(migration *virtv1.VirtualMachineInsta
 				}
 			}
 		case virtv1.MigrationScheduling:
+			if isPodPendingUnschedulable(pod) {
```

Comment:
Hm, this already happens for pods. If you check your pod events you will see that the scheduler gives you events for this already. I think we would have to put the warning on the Migration object.

Reply:
that's what he is doing here, raising an event on the migration object itself when the target pod is unschedulable.

Reply:
wouldn't it be better to just add this even to `handlePendingPodTimeout`? We already check `isPodPendingUnschedulable` there.

Reply:
Seems so I could have sworn that I have read a pod being passed in. 👍

Reply:
@vladikr Hi. Thank you for the suggestion! Moved it there